tcases-lib/src/main/java/org/cornutum/tcases/io/SystemTestDocReader.java

```diff
@@ -194,8 +202,7 @@ public String toIdPath( String attributeName, String attributeValue) throws SAXE
      */
     public String getAttribute( Attributes attributes, String attributeName)
       {
-      String value = attributes.getValue( attributeName);
-      return StringUtils.isBlank( value)? null : value;
```

Comment:
a single whitespace seems like an important boundary test case value to me, for testing various functions.